ovn-tester/ovn_workload.py

```diff
@@ -494,13 +494,13 @@ def add_ports(self, ports, az=0):
         # Always add port IPs to the address set but not to the PGs.
         # Simulate what OpenShift does, which is: create the port groups
         # when the first network policy is applied.
-        if self.addr_set4:
-            for i, nbctl in enumerate(self.nbctl):
+        for i, nbctl in enumerate(self.nbctl):
```

Comment:
@dceara re-looking at the code, should we add port just to the address add_set{4,6}[az]?

Reply:
To simulate what ovn-k8s does we should add the IPs to address sets in all AZs.

Reply:
@dceara ack, fine in this case